### DISCUSSION TITLE
Is this compatible with the WON version of Half-Life?

### DISCUSSION THREAD

**Discussion by sofakng** [hzqst/MetaHookSv#412] [Q&A] [answered]
Is this compatible with the WON version of Half-Life?

I'm using Win98 and the original Half-Life but I'd like to try using the CaptionMod and possibly MetaAudio.

**Comment by hzqst** [comment, marked as answer]
CaptionMod requires VGUI2 and I don't think VGUI2 is even a thing in WON version of Half-Life.

![image](https://github.com/hzqst/MetaHookSv/assets/12287588/e8e67e3c-3ccd-415c-b6fc-c8ae9f071aaf)

**Reply by sofakng** [reply]
Ahhh, OK.  Thanks for the information :)
